src/camera360/camera360.gdshader

```diff
@@ -86,7 +86,18 @@ vec3 fisheye_ray(vec2 p) {
 	float scale = fisheye_forward(vec2(0.0, radians(fovx) / 2.0)).x;
 	return fisheye_inverse(p * scale);
 }
-
+vec3 fulldome(vec2 p) {
+	// This will dezoom enough that we should get exactly
+	// all of the fov in the viewport
+	vec2 dezoomed = p*2.0;
+	// For fulldome output, we want to crop the ouput so that we only render
+	// the circle corresponding to the fov
+	if (length(dezoomed) > 1.0) {
+		return vec3(0.0, 0.0, 0.0);
+	}
+	// If we are within the 1 unit circle, we want the fisheye perspective.
+	return fisheye_ray(dezoomed);
+}
```

Comment:
For consistency, if you could add an empty line after this (I "grouped" the functions by projection, so it makes sense fulldome goes with fisheye).